ports/riptale/port.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "riptale.zip",
+  "items": [
+    "Riptale.sh",
+    "riptale/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Riptale",
+    "porter": [
+      "Slayer366",
+      "MadShmupper"
+    ],
+    "desc": "Riptale is a challenging 2D roguelike action game aiming to make sword combat be as bloody and impactful as possible. The player collects gems that allow them to execute different attacks in the order of their choosing. All this results in a symphony of combos that fill the screen with blood and guts.",
+    "desc_md": null,
+    "inst": "Purchase from Steam https://store.steampowered.com/app/616000/Riptale/ Using only the latest Steam or Windows version, locate game.unx/data.win, audiogroup1.dat & audiogroup2.dat, plus all 8 folders- cavern_levels, challenge_rooms, crypt_levels, establishments, fonts, forest_levels, other, & ruin_levels. Place each of those files and folders inside the gamedata folder.\n   (Note: no need to rename game.unx)\n",
```

Comment:
The inst field is only shown in the PortMaster app, has limited view for instructions. So putting full links in there might not b the best idea.

The inst_md field on the other hand is for the Instructions on the game details page where you can explain as long as you like. 

So on app make it short and snappy and on the website the full blown thing.